### DISCUSSION TITLE
wayland support - possible?

### DISCUSSION THREAD

**Discussion by tigerinus** [fvwmorg/fvwm3#823] [General]
I've got a slow netbook looking for a lightweight efficient desktop environment.

I find that WMs that don't support wayland is often laggy than those do (the effect feels like shaking a jelly) , probably because the CPU cycles involved when communicating betwee client and server in Xorg, which are saved in wayland implementation.

Not sure if it is too early or too late for FVWM3, but still want to bring this up and see if there any thoughts.

Thanks!

**Comment by lgsobalvarro** [comment]
No. FVWM3 relays to heavily on Xlib. Other window managers will arise in time that will mimic some (but most likely all) of FVWM features.

**Comment by ThomasAdam** [comment]
See:  https://github.com/fvwmorg/fvwm3/discussions/1258
